content/en/docs/zero-code/java/spring-boot-starter/out-of-the-box-instrumentation.md

```diff
@@ -118,6 +124,14 @@ public MyService(RestClient.Builder restClientBuilder) {
 }
 ```
 
+As it's possible with the
+[Java agent](/docs/zero-code/java/agent/configuration), you can configure the
+capture of:
+
+- HTTP request and response headers
```

Comment:
this will move to a different page with https://github.com/open-telemetry/opentelemetry.io/pull/4811